.ci/build-freebsd.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -ex
+
+# Pull all the submodules except llvm
+# Note: Tried to use git submodule status, but it takes over 20 seconds
+# shellcheck disable=SC2046
+git submodule -q update --init $(awk '/path/ && !/llvm/ { print $3 }' .gitmodules)
+
+# XXX Drop after Travis upgrades FreeBSD to 12.2 (see also .ci/install-freebsd.sh)
+case $(${CXX:-c++} --version) in
+    *version\ 8.0.*)
+	fetch https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/libcxx-10.0.0.src.tar.xz
+	tar xf libcxx-10.0.0.src.tar.xz
+	export CC=clang10 CXX=clang++10
+	export CXXFLAGS="$CXXFLAGS -nostdinc++ -isystem $PWD/libcxx-10.0.0.src/include"
+	;;
+esac
+
+CONFIGURE_ARGS="
+	-DCMAKE_C_COMPILER_LAUNCHER=ccache
+	-DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+	-DWITH_LLVM=OFF
+	-DUSE_PRECOMPILED_HEADERS=OFF
+	-DUSE_NATIVE_INSTRUCTIONS=OFF
+	-DUSE_SYSTEM_FFMPEG=ON
+	-DUSE_SYSTEM_CURL=ON
+	-DUSE_SYSTEM_LIBPNG=ON
+"
+
+cmake -B build -G Ninja $CONFIGURE_ARGS
```

Comment:
Shellcheck complains about this line, please address it. Other than that, this is an exciting PR.